pkg/network/setup/podnic.go

```diff
@@ -172,6 +179,13 @@ func (l *podNIC) sortIPsBasedOnPrimaryIP(ipv4, ipv6 string) ([]string, error) {
 }
 
 func (l *podNIC) discoverAndStoreCache() error {
+	ifaceLink, err := link.DiscoverByNetwork(l.handler, l.vmi.Spec.Networks, *l.vmiSpecNetwork)
+	if err != nil {
+		log.Log.Reason(err).Errorf("failed to get a link for interface: %s", l.vmiSpecNetwork.Name)
```

Comment:
The virt-handler controller will eventually log it, the is no need for another log here.

Reply:
Done